types/office-js/index.d.ts

```diff
@@ -28518,7 +28517,7 @@ declare namespace Excel {
      *
      * [Api set: ExcelApi 1.8]
      *
-     * @param base64File Optional. The Base64-encoded .xlsx file. The default value is null.
+     * @param base64File Optional. The base64-encoded .xlsx file. The default value is null.
```

Comment:
Looks like these changes reverted back. I don't think it's a big deal to leave it until the changes I made on 10/14 make it in.

Reply:
Ok, thanks! No, the 10/14 changes are not in the source yet.